tests/FakeItEasy.Tests/Core/DefaultFixtureInitializerTests.cs

```diff
@@ -132,7 +132,8 @@ public void Should_set_fake_from_sut_initializer_callback_when_available()
             var fake = A.Fake<IFoo>();
 
             A.CallTo(() => this.sutInitializer.CreateSut(A<Type>._, A<Action<Type, object>>._))
-                .Invokes(x => x.GetArgument<Action<Type, object>>(1).Invoke(typeof(IFoo), fake));
+                .Invokes(x => x.GetArgument<Action<Type, object>>(1).Invoke(typeof(IFoo), fake))
+                .Returns(null);

```

Comment:
Without the `Returns`, our new `Invokes` behaviour returns a fake `object`, which is incompatible with the expected return type.